renderdoc/driver/metal/metal_command_buffer.cpp

```diff
@@ -104,7 +104,7 @@ void WrappedMTLCommandBuffer::commit()
     bool capframe = IsActiveCapturing(m_State);
     if(capframe)
     {
-      record->AddRef();
```

Comment:
this was not intentional and it should have been `bufferRecord`
preventing accidental mistakes like this in the future is the major driving force for this PR